sources/packages/web/src/components/generic/ModalDialogBase.vue

```diff
@@ -59,6 +64,10 @@ export default {
       type: Number,
       required: false,
     },
+    minWidth: {
```

Comment:
I left another comment about this one. Can you please elaborate on why we need it from now on?

Reply:
I assume when the model contents are not lengthy, the modal looks way too narrow/thin. It must be for that.
Correct me if that is wrong @guru-aot

Reply:
Yes @dheepak-aot you are right, I think we should either have min or max width, so removed maxwidth here